src/frontend/enzymes.jl

```diff
@@ -129,9 +132,9 @@ function enzyme_constrained_flux_balance_constraints(
         isozyme_gene_product_stoichiometry,
         gene_product_molar_mass,
         capacity_limits = capacity isa Real ?
-                          [(:total_capacity, gene_ids, C.Between(0, capacity))] :
+                          [(:total_capacity, gene_ids, C.BetweenT(zero(T2), capacity))] :
```

Comment:
you can avoid capturing the T2 by just using the `similar`-based `zero(capacity)`